src/components/game/components/board/components/forecast/Forecast.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+
+const Conditions = (props) => {
+  return (
+    <div>
+      {props.responseObj.cod === 200 ? (
+        <div>
+          <p>
+            Here in {props.responseObj.name} it is currently{" "}
+            {Math.round(props.responseObj.main.temp)} degrees out with{" "}
+            {props.responseObj.weather[0].description}.
+          </p>
+          {Math.round(props.responseObj.main.temp) <= 16 ? (
+            <div>
+              <p>
+                Grab some hot chocolate and play Poker because it's fucking cold
```

Comment:
Remove "fucking" in case of code official inspection from others

Reply:
Fixed, deleting this comment just in case.